src/biome/text/modules/heads/token_classification.py

```diff
@@ -66,25 +89,74 @@ def __init__(
         self.f1_metric = SpanBasedF1Measure(
             self.backbone.vocab,
             tag_namespace=vocabulary.LABELS_NAMESPACE,
-            label_encoding=label_encoding,
+            label_encoding=self._label_encoding,
         )
 
         self.__all_metrics = [self.f1_metric]
         self.__all_metrics.extend(self.metrics.values())
 
+    def _convert_span_labels_to_tag_labels(self, labels: List[str]) -> List[str]:
```

Comment:
I'd like to move this and all methods related to entities conversion to a utils somewhere. The reason is I am working on a relation classifier, which will also receive stand-off annotations as a feature. Potentially other heads will benefit from having entities as features.

Reply:
Done